rust/osm2lanes/src/lib.rs

```diff
@@ -143,6 +250,33 @@ impl LanePrintable for LaneDirection {
     }
 }
 
+impl MarkingStyle {
+    pub fn as_utf8(&self) -> char {
+        match self {
+            Self::SolidLine => '|',
+            Self::BrokenLine => '¦',
+            Self::DashedLine => ':',
+            Self::DottedLine => '᛫',
+            Self::KerbDown => '\\',
+            Self::KerbUp => '/',
+            _ => '|',
```

Comment:
Tradeoff: wildcard matching makes adding a future case a little harder, because the compiler wont force you to think about all of the places the value is used. But that's not so bad for this debug text rendering

Reply:
agreed